ArduPlane/quadplane.h

```diff
@@ -116,6 +116,9 @@ class QuadPlane
 
     // return true if the wp_nav controller is being updated
     bool using_wp_nav(void) const;
+
+    // return true if the user has set ENABLE
+    bool enabled(void) const { return enable != 0; }
```

Comment:
Do we *really* need an accessor at this point in time?

Reply:
I needed someway to poke the private variable :) You can't just poke `plane.quadplane.enable` because from the AP_Arming context `enable` is still private.

```
../../ArduPlane/AP_Arming.cpp: In member function ‘virtual bool AP_Arming_Plane::pre_arm_checks(bool)’:
../../ArduPlane/AP_Arming.cpp:56:25: error: ‘AP_Int8 QuadPlane::enable’ is private within this context
     if (plane.quadplane.enable && !plane.quadplane.available()) {
                         ^~~~~~
compilation terminated due to -Wfatal-errors.
```